Extensions/AdvancedWindow/electron-advancedwindowtools.js

```diff
@@ -267,7 +267,7 @@ gdjs.evtTools.advancedWindow.isAlwaysOnTop = function () {
  */
 gdjs.evtTools.advancedWindow.setPosition = function (x, y) {
   if (gdjs.evtTools.advancedWindow.electronBrowserWindow)
-    gdjs.evtTools.advancedWindow.electronBrowserWindow.setPosition(x, y);
+    gdjs.evtTools.advancedWindow.electronBrowserWindow.setPosition(parseInt(x), parseInt(y));
```

Comment:
Always specify the radix parameter (i.e: 10) to parseInt to avoid weird surprise when someone enter something like "0123".